.github/pull_request_template.md

```diff
@@ -6,7 +6,7 @@
 
 **PR Checklist:**
 
-- [ ] Code is formatted (run `scripts/format`)
+- [ ] Code is formatted (run `pre-commit run --all-files`)
```

Comment:
If my other comment from `.pre-commit-config.yaml` is correct, then this would no longer reformat the code. Maybe we should just provide a direct `black` command here that developers can use.

Reply:
No longer relevant given the decision in [this comment](https://github.com/stac-utils/pystac/pull/401#discussion_r650903004).